src/platform/qt/main.cpp

```diff
@@ -109,8 +109,8 @@ int main(int argc, char* argv[]) {
 #endif
 
 	QTranslator qtTranslator;
-	qtTranslator.load(locale, "qt", "_", QLibraryInfo::location(QLibraryInfo::TranslationsPath));
-	application.installTranslator(&qtTranslator);
+	if (qtTranslator.load(locale, "qt", "_", QLibraryInfo::location(QLibraryInfo::TranslationsPath)))
+		application.installTranslator(&qtTranslator);
```

Comment:
Please follow existing style conventions.

```suggestion
	if (qtTranslator.load(locale, "qt", "_", QLibraryInfo::location(QLibraryInfo::TranslationsPath))) {
		application.installTranslator(&qtTranslator);
	}
```

Reply:
done